src/app/globals.css

```diff
@@ -1,39 +1 @@
-:root {
-    --background: #ffffff;
-    --foreground: #171717;
-}
-
-@media (prefers-color-scheme: dark) {
-    :root {
-        --background: #0a0a0a;
-        --foreground: #ededed;
-    }
-}
-
-html,
-body {
-    max-width: 100vw;
-    overflow-x: hidden;
-}
-
-body {
-    color: var(--foreground);
-    background: var(--background);
-}
-
-* {
-    box-sizing: border-box;
-    padding: 0;
-    margin: 0;
-}
-
-a {
-    color: inherit;
-    text-decoration: none;
-}
-
-@media (prefers-color-scheme: dark) {
-    html {
-        color-scheme: dark;
-    }
-}
+@layer reset, base, tokens, recipes, utilities;
```

Comment:
Just have a knowledge question for myself of what this is doing 🤔

Reply:
yah I did as well, it was in response to this: https://mantine.dev/styles/mantine-styles/#css-layers

Seems like it's needed if you want your styles to override Mantine.  Which we don't currently do, but thought it'd be good to move to it in case we do in the future and then are wondering why it doesn't work.

Reply:
makes sense! sweet